daml-foundations/daml-ghc/daml-prim-src/DA/Types.daml

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+
+daml 1.2
+
+-- | A module containing all the standard types from the base libraries,
+--   so they have nice names when used from Java or similar.
+module DA.Types where
```

Comment:
My biggest question was should this be DA.Type or DA.Types... I went with the latter but open to opposing opinions.

Reply:
I think `DA.Types` is good. `DA.Type` would sound like there's one type called `Type` in there.